.gitignore

```diff
@@ -0,0 +1,3 @@
+packaging/.*
+packaging/builds/*
+packaging/targets/*/debian/changelog
```

Comment:
I think we will need the changelog to be committed. Else when someone builds the package the version will be incorrect. We just shouldn't commit changes to the changelog unless we are releasing to the PPA.

I think just ignore anything in packaging/builds